packages/sdk/src/interaction/interaction.js

```diff
@@ -138,6 +138,16 @@ const makeIx = wat => ix => {
   return Ok(ix)
 }
 
+const prepAccountKeyId = acct => {
+  if (!acct.keyId) return acct
+
```

Comment:
Might be worth adding a some additional invariants to ensure `acct.keyId` is `string` or `number` before processed into number below.

To prevent against:
```javascript
acct.keyId = true

+acct.keyId === 1

acct.keyId = false

+acct.keyId === 0
```

Reply:
Good catch!  keyId = 0 is falsy which makes this no beuno